jbmc/src/java_bytecode/replace_java_nondet.cpp

```diff
@@ -92,7 +92,7 @@ get_nondet_instruction_info(const goto_programt::const_targett &instr)
   {
     return nondet_instruction_infot();
   }
-  const auto &code = to_code(instr->code);
+  const auto &code = instr->code;
```

Comment:
The code should be reviewed/cleaned up a bit more: why is the `instr->code.id()` above necessary? The `get_statement` below this one should be an invariant as `instr->is_function_call()` has been checked above.